app/products/calls/simple-peer.ts

```diff
@@ -103,18 +103,24 @@ export default class Peer extends stream.Duplex {
             this.onFinish();
         };
 
+        const connConfig = {
+            iceServers: [
+                {
+                    urls: [
+                        'stun:stun.l.google.com:19302',
+                        'stun:global.stun.twilio.com:3478',
```

Comment:
I guess this is temporary, right ?

Reply:
Those are the defaults if nothing is configured on the plugin. Eventually we'll be hosting our own public STUN server and we'll be able to replace those.